docs/contribution_handling.md

```diff
@@ -0,0 +1,53 @@
+# WinUI Repo Contribution Handling
+
+The WinUI repo is intended both as a place for the WinUI community to discuss development with each other, and as a place to provide insight into upcoming changes and bug fixes the WinUI team is working on before updates are released. Here, we’ll outline how we handle feature requests and bugs that the community opens.
+
+## Issues
+
+Feature requests and bugs are tracked as GitHub issues.
+
+For reporting security issues please see the [Security Policy](SECURITY.md).
+
+For all other bugs and general issues please [file a new issue](https://github.com/Microsoft/microsoft-ui-xaml/issues/new/choose) using the Bug Report template.
+
+Please file questions and discussions as discussions, not issues.
+
+## Feature Proposals
+
+Feature proposals for WinUI 3 are categorized based on the likely timeframe in which the feature team can consider them.
+
+1. 	Short-term, less than a year
+2.	Long-term, not likely in the next year
+
+All feature proposals are automatically categorized as long-term and unlikely to be considered, although the WinUI team might change the priority based on customer and business needs. If the WinUI team does close a feature proposal, we will indicate the reason why and how it fits into our plans. Closing a proposal does not mean that the team will never consider it. We encourage the community to add comments or interact with a proposal at any time, even a closed one, to signal its importance to the team.
+
+### Feature Planning
+
+The WinUI team plans on sharing what’s coming in the next major release once the feature list is ready, although due to frequently changing business priorities we are not committed to a long-term roadmap. Additionally, preview and experimental releases provide opportunities for the community to see what’s new and what’s changing on a more frequent basis.
+
+For more info about the what's coming in the next WinUI 3 release, see the [Windows App SDK feature roadmap](https://github.com/microsoft/WindowsAppSDK/blob/main/docs/roadmap.md).
+
+## Bugs
+
+### WinUI 2
+
+Because WinUI 2 is in maintenance mode, new bugs filed against it are closed unless they’re a security issue or are business critical. 
```

Comment:
I feel like I'm missing something, how does "being in maintenance mode" entail that all bugs (including regressions??) will just be closed unless they're a security issue or are business critical? I'm very confused here for two reasons:
- For UWP XAML control bugs (ie. not strictly WinUI 2), those would just be OS bugs, so I don't see how they can just be ignored. Can we clarify here whether this line _strictly_ only refer to WinUI 2 controls and not built-in ones? Additionally, if so, then can we eg. explain the answers being given in #6513? Since that's a bug that's affecting a built-in UWP XAML control.
- For WinUI 2 controls, again, why should bugs be closed directly? I fully understand that the project is in maintenance mode, but doesn't "maintenance" include fixing bugs? I do get that they might not be a priority, sure, but why just say they would never be fixed at all?

Additionally, could we further clarify how community contributions are approached? There's lots of folks in the community that would be willing to help fixing bugs and whatnot, and once again I do get that the project is only in maintenance mode, but wouldn't that still allow contributions to (slowly) be reviewed and accepted if sufficient?

I mean I'm thinking about Win2D as a point of reference, which is also effectively in maintenance mode, but we still occasionally add some new features if there's contributions for them, and we also happily accept contributions for bug fixes, even though it might take a while to find time to review them.

Reply:
Thanks for the detailed feedback, Sergio! Since the phrase "maintenance mode" at the root of your confusion doesn't accurately convey the team's intentions with WinUI 2 bugs, I've updated the wording there. As for bugs in the OS database, those are handled per the norm for those bugs. I know the updated wording doesn't address every nuance of what you're asking here, but if you'd like to get into more of an internal bug discussion then please feel free to spin up a thread with myself and the WinUI dev leads. :) Thanks!

Reply:
> if you'd like to get into more of an internal bug discussion then please feel free to spin up a thread with myself and the WinUI dev leads

I encourage you to please keep it in the public community space. Sergio's issues might affect many of us

Reply:
Hey @duncanmacmichael, thank you for discussing this internally and updating the readme! That is much better. However I still have a couple questions that I'd like to get more clarity on. Note: take this as me just being some indie dev from the community that loves making apps and libraries for Windows, not as me being a Microsoft engineer.
- Regarding OS bugs, that's good to here, and in line with what I expected. Though I will  point out, it's _very_ difficult for many folks in the community to understand exactly where the line is between what is "OS stuff" and what is "WinUI stuff". Effectively speaking, this repo has historically been a place for people to report issues with both, as in, including for System XAML controls and other generic issues in that area. If we're saying we're just going to close issues related to those now, what is the place for people to report them and track them? Sure, I can just log an internal bug in the OS repo and ping folks, but others in the community can't do that (and even then, I would like my bug reports to be in the open as much as possible, so that they can also be referenced by others if needed, etc.). Do we have a good solution for this?
- It's totally fair that the team is mostly focused on WinUI 3, though (maybe in the near future, not right now), it would be awesome if we could go back and reconsider at least a general story for contributions to WinUI 2. As I mentioned, there's lots of small bugs that people would love to contribute fixes to, even though they're not technically "business critical". Similarly, especially once WinUI 3 goes open source, there's several community developers that have already said they'd love to take on the work to backport eg. the new controls to WinUI 2. Would it be possible for us to figure out a way to not shut down those kinds of contributions entirely? As in, even just having a very small amount of work going into that, so that WinUI 2 can keep accepting contributions (even slowly, just not completely halted) would be amazing. I also personally think keeping at least the XAML controls in sync between WinUI 2 and WinUI 3 would only benefit the ecosystem, and you can see angle for this for WinUI 3 as well: it would allow UWP developers to more easily migrate to WinUI 3 too if they wanted, as they could start adopting new controls soon without having to pay all the huge cost upfront of swapping the entire framework. And it'd also fit in quite nicely in the whole "meet developers where they are" motto.

Just sharing some thoughts here. I personally would love to see WinUI 2 and WinUI 3 happily cohexist.
Thank you for listening 🙂

P.S. this was more of a general thing, of course if we were to hit a specific bug that was critical for us in the Microsoft Store, I'd report that internally as you suggested. But I'd love for this kind of general conversation to continue being in the open otherwise. As I said, just sharing my thoughts as any other community member passionate about Windows development 😄

Reply:
> > if you'd like to get into more of an internal bug discussion then please feel free to spin up a thread with myself and the WinUI dev leads
> 
> I encourage you to please keep it in the public community space. Sergio's issues might affect many of us

Totally understood, and apologies for that wording. Again :D Any issues opened in the repo or are otherwise in the public sphere will remain so, and we have no intention of trying to cut that off. I was trying to articulate that there are always going to be some things that have to be kept internal, such as implementation details of OS bug fixes, and I was encouraging him to talk to us about things in addition to the public bug discussions. 

Hope this clarifies things a bit!

Reply:
> Hey @duncanmacmichael, thank you for discussing this internally and updating the readme! That is much better. However I still have a couple questions that I'd like to get more clarity on. Note: take this as me just being some indie dev from the community that loves making apps and libraries for Windows, not as me being a Microsoft engineer.
> 
> * Regarding OS bugs, that's good to here, and in line with what I expected. Though I will  point out, it's _very_ difficult for many folks in the community to understand exactly where the line is between what is "OS stuff" and what is "WinUI stuff". Effectively speaking, this repo has historically been a place for people to report issues with both, as in, including for System XAML controls and other generic issues in that area. If we're saying we're just going to close issues related to those now, what is the place for people to report them and track them? Sure, I can just log an internal bug in the OS repo and ping folks, but others in the community can't do that (and even then, I would like my bug reports to be in the open as much as possible, so that they can also be referenced by others if needed, etc.). Do we have a good solution for this?
> * It's totally fair that the team is mostly focused on WinUI 3, though (maybe in the near future, not right now), it would be awesome if we could go back and reconsider at least a general story for contributions to WinUI 2. As I mentioned, there's lots of small bugs that people would love to contribute fixes to, even though they're not technically "business critical". Similarly, especially once WinUI 3 goes open source, there's several community developers that have already said they'd love to take on the work to backport eg. the new controls to WinUI 2. Would it be possible for us to figure out a way to not shut down those kinds of contributions entirely? As in, even just having a very small amount of work going into that, so that WinUI 2 can keep accepting contributions (even slowly, just not completely halted) would be amazing. I also personally think keeping at least the XAML controls in sync between WinUI 2 and WinUI 3 would only benefit the ecosystem, and you can see angle for this for WinUI 3 as well: it would allow UWP developers to more easily migrate to WinUI 3 too if they wanted, as they could start adopting new controls soon without having to pay all the huge cost upfront of swapping the entire framework. And it'd also fit in quite nicely in the whole "meet developers where they are" motto.
> 
> Just sharing some thoughts here. I personally would love to see WinUI 2 and WinUI 3 happily cohexist. Thank you for listening 🙂
> 
> P.S. this was more of a general thing, of course if we were to hit a specific bug that was critical for us in the Microsoft Store, I'd report that internally as you suggested. But I'd love for this kind of general conversation to continue being in the open otherwise. As I said, just sharing my thoughts as any other community member passionate about Windows development 😄

Glad the updated wording helped, Sergio! And I totally get it, on GitHub you're Sergio the dev enthusiast, not the MS employee. :) I really do appreciate your thoughtful feedback here and will take up the WinUI 2 story with the dev team when it's appropriate. To be transparent we don't have the cycles for it right now since we're busy planning the next release, but never say never, right? So I won't be updating this wording anymore for now, but again, thanks!